website/static/js/filepage/revisions.js

```diff
@@ -30,6 +33,7 @@ var FileRevisionsTable = {
         self.enableEditing = enableEditing;
         self.baseUrl = (window.location.href).split('?')[0];
 
+        model.hasHashes = self.file.provider === 'osfstorage';
```

Comment:
Rather than checking against addon name, can you instead check against the existence of the `md5` and `sha256` metadata? Keep in mind we may support hashes for other addons in the future.